python/ray/tune/__init__.py

```diff
@@ -22,15 +22,6 @@
 from ray.tune.trainable import Trainable
 from ray.tune.callback import Callback
 from ray.tune.search import grid_search
-from ray.tune.trainable.session import (
```

Comment:
Are these deprecation related to lightgbm/xgboost or just d it casually?

Reply:
This is due to `xgboost_ray` having a dependency on `ray.tune.trainable.session.is_session_enabled`. That was blocking this module from being removed, and it's already been hard-deprecated for a few releases. Now that the external dependency is gone, we can delete.